pkg/registry/apiserverinternal/storageversion/storage/storage.go

```diff
@@ -81,3 +84,8 @@ func (r *StatusREST) Update(ctx context.Context, name string, objInfo rest.Updat
 	// subresources should never allow create on update.
 	return r.store.Update(ctx, name, objInfo, createValidation, updateValidation, false, options)
 }
+
+// GetResetFields implements rest.ResetFieldsStrategy
```

Comment:
This function name is not very clear. "Reset" is a verb at least as often as a noun which makes it especially confusing.

What is the difference between "reset" and "disabled" fields?

Reply:
In this case "Reset" is an adjective. A reset field is a field that should be cleared from the field managers set of fields that it owns.

We've been looking for a better name for over a year and haven't found one I guess https://github.com/kubernetes/enhancements/pull/1123/files#r356309605 

A disabled field is a field on an object used by a feature flag that is disabled.

Reply:
Also not happy with the naming, I remember we talked about this in the past (like a year ago) but didn't find something better. In SMD it's called ignoredFields right now as it suits the specific context more, but ignoredFields does not really fit here I think.